README.md

```diff
@@ -72,22 +72,29 @@ the build speed and development experience by using Vite and Turborepo.
 
 ## Getting started
 
-1. When you're using Windows run this:
-    - `git config --global core.eol lf`
-    - `git config --global core.autocrlf input`
-
-   **This will set the EOL (End of line) character to be the same as on Linux/macOS. Without this, our bash script won't
-   work, and you will have conflicts with developers on Linux/macOS.**
-2. Clone this repository.( ```git clone https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite``` )
-3. Ensure your node version is >= than in `.nvmrc` file, recommend to
-   use [nvm](https://github.com/nvm-sh/nvm?tab=readme-ov-file#intro)
-4. Edit `/packages/i18n/locales/`{your locale(s)}/`messages.json`
-5. In the objects `extensionDescription` and `extensionName`, change the `message` fields (leave `description` alone)
-6. In `/.package.json`, change the `version` to the desired version of your extension.
-7. Install pnpm globally: `npm install -g pnpm` (ensure your node version >= 22.12.0)
-8. Run `pnpm install`
-
-Then, depending on the target browser:
+1. Clone this repository.( ```git clone https://github.com/Jonghakseo/chrome-extension-boilerplate-react-vite``` )
+2. Ensure your node version is >= than in `.nvmrc` file, recommend to use [nvm](https://github.com/nvm-sh/nvm?tab=readme-ov-file#intro)
+3. Edit `/packages/i18n/locales/`{your locale(s)}/`messages.json`
+4. In the objects `extensionDescription` and `extensionName`, change the `message` fields (leave `description` alone)
+5. In root `package.json`, change the `version` to the desired version of your extension.
+6. Install pnpm globally: `npm install -g pnpm`
+7. Run `pnpm install`
+8. Check if you have these settings in your IDE/Editor:
+    - <b>VSCode</b>:
+        - Installed [ESLint extension](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
+        - Installed [Prettier extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode)
+        - Enabled `Typescript Workbench version` in settings:
+            - CTRL + SHIFT + P -> Search: `Typescript: Select Typescript version...` -> `Use Workbench version`
+    - <b>WebStorm</b>:
+      - Installed [Prettier plugin](https://plugins.jetbrains.com/plugin/10456-prettier)
+      - Enabled `File | Settings | Languages & Frameworks | JavaScript | Code Quality Tools | ESLint`\
+      -> `Automatic ESLint configuration` and `Run eslint --fix on save`
+      - Enabled `File | Settings | Languages & Frameworks | JavaScript | Prettier`\
+      -> `Automatic Prettier configuration` and `Run on save`
+      - Optional, but useful `File | Settings | Tools | Actions on Save`\
+      -> `Optimize imports` and `Reformat code`
+
+<b>Then, depending on the target browser:</b>
```

Comment:
Hmm... I'm wondering if discussing IDE settings might be out of scope for this boilerplate. 

I'm not sure if it's appropriate to rely on the UI of a specific IDE like `VSCode` or `WebStorm`. 
I'm also worried that in the future, people might start adding instructions for other IDEs to the README.

Reply:
Maybe not, but take a look, we have husky, which running eslint and prettier before commit and because of that i think it's good to have support with these actions from IDE, and it can help VSC to recognize TS version of project.

Maybe we could place it in other place/file.

Have you any idea?

Reply:
Well... in any case, being considerate can be helpful for someone, so I think it's good. However, the plugin links and menu descriptions here are inevitably tied to the IDE’s UI. I’m starting to wonder if it’s really necessary to go into this much detail. 

If there’s official documentation for these settings, how about just leaving a link to that instead?

Reply:
Ok, there's
I had been thinking about that and i've forgot about that.
I'll change it in minutes :)

Reply:
@Jonghakseo Ok, done.
There's not official docs about setting TS version in VSC, only stackoverflow thread, that's why i've decided to leave it like that.

Rest have been simplified, i hope it looks better now :)